client/go/cmd/test.go

```diff
@@ -0,0 +1,100 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+// vespa test command
+// Author: jonmv
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/vespa-engine/vespa/client/go/vespa"
+	"log"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	endpointsFlag     = "endpoints"
+	dataPlaneCertFlag = "data-plane-public-cert"
+	dataPlaneKeyFlag  = "data-plane-private-key"
+)
+
+var (
+	endpointsArg     string
+	dataPlaneCertArg string
+	dataPlaneKeyArg  string
+)
+
+func init() {
+	rootCmd.AddCommand(testCmd)
+	testCmd.PersistentFlags().StringVar(&endpointsArg, endpointsFlag, "", "The endpoints to use, for each container cluster")
```

Comment:
Not sure I understand why this is an option. Shouldn't this discover relevant endpoints automatically?

Reply:
When run by a developer, yes.
When run in the tester-container, endpoints are provided as config, and we can't ask the controller. 

I wonder, though, if these overrides all should be ENV instead?

Reply:
> When run in the tester-container, endpoints are provided as config, and we can't ask the controller.

Right.

> I wonder, though, if these overrides all should be ENV instead?

Good idea. That what we use for overriding system today, i.e. things regular users don't typically care about.